### PR TITLE
Bug 1816773: add information about the possibility to remove UsingDep…

### PR DESCRIPTION
**Pull request by bergerhoffer**
…recatedAPIExtensionsV1Beta1 alert

Replaces #20878.

https://bugzilla.redhat.com/show_bug.cgi?id=1816773

Preview: http://file.rdu.redhat.com/~ahoffer/2020/pr-20878-update/monitoring/cluster_monitoring/managing-cluster-alerts.html

### CODE REVIEW

File: modules/monitoring-known-issues.adoc [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+** If `UsingDeprecatedAPIExtensionsV1Beta1` is the only rule listed in the output, remove it using the following command:
++
+----
+$ oc delete prometheusrules.monitoring.coreos.com/kube-apiserver -n openshift-kube-apiserver
+----
+
+** If there are more rules, you must manually remove only the `UsingDeprecatedAPIExtensionsV1Beta1` rule from that resource.
++
+// TODO: Using oc edit on prometheusrules.monitoring.coreos.com/kube-apiserver?
```
Comment (bergerhoffer) [inline modules/monitoring-known-issues.adoc:38]: @soltysh Can you confirm if they would do `oc get prometheusrules.monitoring.coreos.com/kube-apiserver -n openshift-kube-apiserver` to manually remove the `UsingDeprecatedAPIExtensionsV1Beta1` rule? If so, I'll just add this here for consistency in the procedure.

**Comment by sichvoge** [comment]
Can someone give me a td;lr on what this is about and why someone needs to manually remove it?

File: modules/monitoring-known-issues.adoc (hunk 1 above)
Comment (soltysh) [reply modules/monitoring-known-issues.adoc:38]: You mean `oc edit` - yes, it should work.

Comment (soltysh) [reply modules/monitoring-known-issues.adoc:38]: Even if users will remove more than needed, the operator will fix the missing bits back.

**Review by soltysh — APPROVED** [review]
lgtm

**Comment by bergerhoffer** [comment]
> Can someone give me a td;lr on what this is about and why someone needs to manually remove it?

@soltysh can you give a quick background on this alert please?

**Comment by soltysh** [comment]
> @soltysh can you give a quick background on this alert please?

The alert was added in 4.3 to inform the users of deprecated APIs they are using. Initially we were filtering out a set of clients which although using those APIs were not to be addressed by the cluster administrator. Unfortunately, the ability to filter those based on clients was removed due to a bug in the code and we were not able filter out the aforementioned clients. This would lead to overflowing any cluster with alerts and would only cause confusion to users. That's why we've decided to drop this alert. Hope that helps :smile:

**Comment by soltysh** [comment]
@bergerhoffer can we push this forward?

**Comment by bergerhoffer** [comment]
@sichvoge Please let us know if you have any concerns with this addition.

@juzhao Can you please review this from a QE perspective? This is for 4.3 only. Thanks!

**Comment by sichvoge** [comment]
Still not sure why you need my blessing for this specific addition :) Shouldn't that be the team who owns the alert?

**Comment by bergerhoffer** [comment]
@sichvoge Mostly just because it's an addition to the monitoring section and @rh-max had asked :). But if you don't have any objections, then we'll get this QE'd and in. Thanks!

**Comment by bergerhoffer** [comment]
@juzhao Can you please review? Or @lihongyan1? Thanks!

Preview: http://file.rdu.redhat.com/~ahoffer/2020/pr-20878-update/monitoring/cluster_monitoring/managing-cluster-alerts.html

**Comment by soltysh** [comment]
ping @juzhao @lihongyan1

**Pull request closed without merging**